src/index.html

```diff
@@ -8,34 +8,34 @@
     <link rel="stylesheet" href="./styles.css">
   </head>
   <body>
-    <section class="user-info">
-      <h1>Hi, I'm John</h1>
-      </section>
     <header class='main-header'>
       <div class="wrapper">
         <img class="hamburger-menu" src="./images/hamburger-icon.png" alt="hamburger icon serving as menu button">
-        <img class="x-menu x" src="./images/x-icon.png" alt="x icon to be clicked when you are done viewing the menu options">
+        <img class="x-menu hidden" src="./images/x-icon.png" alt="x icon to be clicked when you are done viewing the menu options">
       </div>
       <h1 class="header-title">What's Cookin'?</h1>
-      <h1 class="open-menu-title x">Menu</h1>
-      <div class="spacer"></div>
+      <section class="user-info">
+        <p>Hi, I'm John</p>
```

Comment:
this is a friendly reminder to us to remove the filler text so that it doesn't overlap with the `innerText` we show on the DOM